content/learn/vp_agof.adoc

```diff
@@ -0,0 +1,381 @@
+
+.. Update your `agof_vault.yml` file with the path to the downloaded manifest zip file. For example add the following: 
++
+[source,shell]
+----
+controller_license_src_file: '~/Downloads/manifest_<sub_allocation_name>_20240924T131518Z.zip'
+manifest_content: "{{ lookup('file', controller_license_src_file) | b64encode }}"
+----
++
+.Example agof_vault.yml file
+
+[source,yaml]
+----
+---
+aws_account_nbr_vault: '293265215425'
+aws_access_key_vault: 'AKIAIJ6ZIKPAUZGF2643'
+aws_secret_key_vault: 'gMC3Jy3/MZtOosjUDHy0Nl/2mp2HQok1JDfCQGKUR'
+
+pattern_prefix: 'foga'
+pattern_dns_zone: 'aws.validatedpatterns.io'
+
+ec2_name_prefix: 'foga-testing'
+ec2_region: 'us-east-1'
+
+offline_token: '<insert-token-here>'
+
+redhat_username: 'rhn-support-myusername'
+redhat_password: 'passwd01'
+
+admin_password: 'redhat123!'
+
+manifest_content: "Content for a manifest file to entitle AAP Controller. See below for an example of how to point to a local file"
+#manifest_content: "{{ lookup('file', '~/Downloads/manifest_AVP_20230510T202608Z.zip') | b64encode }}"
+
+org_number_vault: "1778713"
+activation_key_vault: "kevs-agof-key"
+
+# Set these variables to provide your own AMI, or to re-use an AMI previously generated with this process
+#skip_imagebuilder_build: 'boolean: skips imagebuilder build process'
+#imagebuilder_ami: 'The ID of an AWS AMI image, preferably one that was built with this toolkit'
+
+automation_hub_token_vault: '<insert-token-here>'
+
+# These variables can be set but are optional. The previous (before AAP 2.4) conncept of sync-list was private
+# to an account.
+#automation_hub_url_certified_vault: 'The private automation hub URL for certified content'
+#automation_hub_url_validated_vault: 'The private automation hub URL for validated content'
+
+controller_config_dir: "{{ '~/agof_minimal_demo/config' | expanduser }}"
+
+db_password: 'test'
+
+agof_statedir: "{{ '~/agof' | expanduser }}"
+
+agof_iac_repo: "https://github.com/mhjacks/agof_demo_config.git"
+----
+
+. Run from the AGOF repository directory the following command : 
++
+[source,terminal]
+----
+$ ./pattern.sh make install
+----
+This command invokes the `controller_configuration` `dispatch` role on the controller endpoint based on the configuration found in the `controller_configuration_dir` and in your `agof_vault.yml` file. This controls all of the controller configuration. Based on the variables defined in the controller configuration, `dispatch` calls the necessary roles and modules in the right order to configure AAP to run your pattern.
+
+.Verification
+
+The default installation provides an AAP 2.4 installation deployed using the containerized installer, with services deployed this way:
+
+.agof_vault settings
+[cols="30%,70%",options="header"]
+|===
+| URL | Service
+
+| `https:{{ ec2_name_prefix }}.{{ domain }}:8443`
```

Comment:
The URLs are all rendering incorrectly here.

They should all be:

https://aap.{{ ec2_name_prefix }}.domain:port number